app/uk/gov/hmrc/customs/emailfrontend/model/LoggedInUser.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.model
+
+import uk.gov.hmrc.auth.core.{EnrolmentIdentifier, Enrolments}
+
+case class LoggedInUser(enrolments: Enrolments, internalId: InternalId) {
+  lazy val eori: Option[EnrolmentIdentifier] = enrolments.getEnrolment("HMRC-CUS-ORG") flatMap (_.getIdentifier("EORINumber"))
+}
+
+case class Eori(id: String)
+object Eori {
+  def apply(identifier: EnrolmentIdentifier) = new Eori(identifier.value)
+}
+
+case class InternalId(id: String)
+object InternalId {
+  def apply(internalId: Option[String]) = new InternalId(internalId getOrElse "")
```

Comment:
Internal Id not available we cannot proceed  I think this will cause an issue we should making this an empty string. In this case our application he is not authenticated